src/libraries/System.Security.Cryptography.X509Certificates/tests/RevocationTests/DynamicRevocationTests.cs

```diff
@@ -17,11 +17,33 @@ public static class DynamicRevocationTests
 
         private static readonly Oid s_tlsServerOid = new Oid("1.3.6.1.5.5.7.3.1", null);
 
+        private static bool SupportsEntireChainCheck => !PlatformDetection.IsAndroid;
+
         private static readonly X509ChainStatusFlags ThisOsRevocationStatusUnknown =
-                PlatformDetection.IsOSX ?
+                PlatformDetection.IsOSX || PlatformDetection.IsAndroid ?
                 X509ChainStatusFlags.RevocationStatusUnknown :
                 X509ChainStatusFlags.RevocationStatusUnknown | X509ChainStatusFlags.OfflineRevocation;
 
+        // Android will stop checking after the first revocation error, so any revoked certificates
+        // after will have PartialChain and RevocationStatusUnknown instead of Revoked
+        private static readonly X509ChainStatusFlags ThisOsRevokedWithPreviousRevocationError =
+                PlatformDetection.IsAndroid ?
+                X509ChainStatusFlags.PartialChain | X509ChainStatusFlags.RevocationStatusUnknown :
```

Comment:
This doesn't make sense to me... If we know something got revoked, why is PartialChain popping up? We built a full chain...

Reply:
Ah, yeah, this was from always setting `PartialChain` after any error. Should be able to fix this, since I have with / without revocation split now.